### PR TITLE
update week4 schedule

### PR DESCRIPTION
**Pull request by shahenazmonia**

### CODE REVIEW

File: coursebook/week-4/README.md [hunk 1]
```diff
@@ -11,72 +11,73 @@
...
 
 ### Day 2
 
-- 10:00 - 11:30
+- 8:30 - 10:00
   -- [ES6 morning challenge](./morning-challenge-day-2.md)
 
-- 11:30 - 13:00
+- 10:00 - 12:00
   -- [Node Girls workshop](https://github.com/node-girls/workshop-cms) (Day 2 of 2)
```
Comment (jema28) [inline coursebook/week-4/README.md:34]: Are we doing all node girls steps on Day 1? What are we replacing it with here?

Comment (shahenazmonia) [reply coursebook/week-4/README.md:34]: No, I just split the workshop to before/after lunch, we have an hour after code along

File: coursebook/week-4/README.md [hunk 2]
```diff
@@ -11,72 +11,76 @@
...
+- 13:30 - 15:30
+    --  (4-6) steps of [Node Girls workshop](https://github.com/jema28/node-workshop) (Day 1 of 2)
+- 15:30 - 16:30
+  -- Typing
 
 ### Day 2
 
-- 10:00 - 11:30
+- 8:30 - 10:00
   -- [ES6 morning challenge](./morning-challenge-day-2.md)
```
Comment (eadehemingway) [inline coursebook/week-4/README.md:32]: ive changed the ES6 workshop can you change the link to this https://github.com/fack2/ES6-workshop
please?

Comment (shahenazmonia) [reply coursebook/week-4/README.md:32]: done

**Pull request merged**